site/.build.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const module_path = './ts/';
```

Comment:
Let's use `path.join` to help ensure this is cross-OS compatible:

```suggestion
const path = require('path');
const module_path = path.join(__dirname, 'ts');
```